app/models/goldencobra/upload.rb

```diff
@@ -99,6 +99,7 @@ def unzip_files
         zipped_files = Zip::File.open(self.image.path)
         zipped_files.each do |zipped_file|
           filename = zipped_file.name.split('/').last.gsub(" ", "_")
+          next if filename[0] == "."
           if zipped_file.file?
             zipped_file.extract("tmp/#{self.id}_#{filename}")
```

Comment:
`"tmp/#{self.id}_#{filename}"` is used three times.
You could store it in a `file_path`?!